### PR TITLE
Allow Chewie controls to be positioned to allow for a larger safe area.

### PR DESCRIPTION
**Pull request by jweidner-mbible**

### CODE REVIEW

**Comment by diegotori** [comment]
LGTM. Once it passes automated checks, then it'll get merged into master.

File: lib/src/chewie_player.dart [hunk 1]
```diff
@@ -525,6 +527,9 @@ class ChewieController extends ChangeNotifier {
   /// Defines a delay in milliseconds between entering buffering state and displaying the loading spinner. Set null (default) to disable it.
   final Duration? progressIndicatorDelay;
 
+  /// allows extra padding to position controls as desired
```
[the remark this answers is not in the document]
Comment (jweidner-mbible) [reply lib/src/chewie_player.dart]: updated doc comment

**Comment by diegotori** [comment]
@jweidner-mbible please re-sync your change with master. Thanks.

**Comment by jweidner-mbible** [comment]
@diegotori I just re-synced my change with master.

**Comment by caitlynbell399** [comment]
@diegotori Any idea on when this might get merged into master? :)

**Comment by diegotori** [comment]
@caitlynbell399 rerunning the lint and format CI tasks. If they're green, then they'll look good to me

**Comment by diegotori** [comment]
@jweidner-mbible looks like the CI checks failed. Please address them, or else I won't be able to merge. 

Thanks.

**Comment by caitlynbell399** [comment]
@diegotori Committed a fix for the issue. Thank you for your help with this :)

**Comment by diegotori** [comment]
LGTM. Merging.

**Pull request merged**